tests/search/schemachanges/schemachanges_need_refeed_reconfig.rb

```diff
@@ -43,6 +43,14 @@ def test_need_refeed_reconfig
     assert_result("sddocname:test&nocache", @test_dir + "result.1.xml")
     assert_hitcount("f1:b&nocache", 2)
     assert_hitcount("f3:%3E29&nocache", 2)
+
+    # redeploy again to trigger reindexing, then wait for up to 5 minutes for document 1 to be reindexed
+    redeploy("test.1.sd")
+    start = Time.now
+    until /<field name="a1">30<\/field>/ =~ search("sddocname:test&nocache") or Time.now - start > 300 # seconds
```

Comment:
I don't think we should rely on regexp matching here. It is better to use the Resultset object and find the hit and field value there.

Reply:
Thanks.